File: result/repair/transcoder/python2java/java_repair/NON_REPEATING_ELEMENT.java
  int f_gold ( int [ ] arr , int n ) {
  for ( int i = 0 ;
  i < n ;
  i ++ ) {
    int j = 0 ;
    while (j<n){
      if ( i != j && arr [ i ] == arr [ j ] ) {
        break ;
      }
      j ++ ;
    }
    if ( j == n ) {
      return arr [ i ] ;
    }
  }
  return - 1 ;
}

File: result/translate/starchat/java2python/python/NON_REPEATING_ELEMENT.py
def f_gold(arr, n):
        for i in range(n):
            j = 0
            while j < n and i!= j:
                if arr[i] == arr[j]:
                    j += 1
                else:
                    j += 1
            if j == n:
                return arr[i]
        return -1